diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/StringTemplateFormatRule.kt

```diff
@@ -77,4 +85,17 @@ class StringTemplateFormatRule(private val configRules: List<RulesConfig>) : Rul
             }
         }
     }
+
+    @Suppress("UnsafeCallOnNullableType")
+    private fun bracesCanBeOmitted(node: ASTNode): Boolean {
+        return if (node.findAllNodesWithSpecificType(REFERENCE_EXPRESSION).size == 1) {
+            node.findAllNodesWithSpecificType(REFERENCE_EXPRESSION).first().treeParent.elementType == LONG_STRING_TEMPLATE_ENTRY
+        } else {
+            node.hasAnyChildOfTypes(FLOAT_CONSTANT, INTEGER_CONSTANT)
+        } && !node.hasAnyChildOfTypes(CALL_EXPRESSION)
+                && (node.treeNext.text.startsWith(" ")
```

Comment:
It can start with '.' or ',' too, because these are not valid symbols for variable name. You could check whether the next symbol is alphanumeric, for example.